rectype,issueid,project_owner,project_name,actor,time,text,action,title,clean_text,Test
rectype,issueid,project_owner,project_name,actor,time,text,action,title,text,N
issue_title,221,pycqa,astroid,pylint-bot,2015-10-21 07:48:28,"Originally reported by: **Claudiu Popa (BitBucket: [PCManticore](http://bitbucket.org/PCManticore), GitHub: @PCManticore)**

---

---
- Bitbucket: https://bitbucket.org/logilab/astroid/issue/221
",start issue,"Test failures on pypy, pypy3 and jython",origin report claudiu popa bitbucket pcmanticor github pcmanticor bitbucket,N
issue_closed,221,pycqa,astroid,PCManticore,2016-06-27 17:35:30,nan,closed issue,"Test failures on pypy, pypy3 and jython",nan,N
issue_comment,221,pycqa,astroid,pylint-bot,2015-11-06 23:51:40,"_Original comment by_ **BitBucket: [ceridwenv](http://bitbucket.org/ceridwenv), GitHub: @ceridwen**:

---

PyPy tests should be clean as of b68ee1186e5f.
",nan,nan,origin comment bitbucket ceridwenv github ceridwen pypi test clean b68ee1186e5f,Y
issue_comment,221,pycqa,astroid,PCManticore,2016-06-27 17:35:30,"PyPy and Jython are again erroring out with the latest astroid. I'm closing this issue, since we already have the errors in the CI (and we should strive to make it green all the time).
",nan,nan,pypi jython error latest astroid Im close issu sinc alreadi error CI strive make green time,N
